src/main/java/com/fcgl/madrid/search/dataModel/UserSearch.java

```diff
@@ -1,24 +1,22 @@
 package com.fcgl.madrid.search.dataModel;
-import org.springframework.data.annotation.Id;
 import java.util.Date;
 
 public class UserSearch {
 
-    @Id
-    public int id;
-    public Integer userId;
+    public Long userId;
     public String query;
     public Date addedOn;
 
-    public UserSearch(int id, String categoryName, String description, Date addedOn, Date lastUpdated) {
-        this.id = id;
+    public UserSearch(Long userId, String query, Date addedOn) {
+        this.userId = userId;
+        this.query = query;
         this.addedOn = addedOn;
```

Comment:
Maybe instead of passing in addedOn we set it here to the current time?

Reply:
I think it makes sense to set time at the database level and just keeping these objects as store.

Reply:
What do you mean by `objects at store`?

I meant doing the `Date date = new Date();` in this constructor instead of having to pass in `addedOn`